xbmc/pvr/filesystem/PVRGUIDirectory.cpp

```diff
@@ -230,6 +231,18 @@ void GetSubDirectories(const CPVRRecordingsPath& recParentPath,
     item->SetLabel2(StringUtils::Format("%s / %s",
         item->GetProperty("watchedepisodes").asString().c_str(),
         item->GetProperty("totalepisodes").asString().c_str()));
+
+    item->IncrementProperty("sizeinbytes", static_cast<double>(recording->GetSizeInBytes()));
```

Comment:
Maybe about them it clould make sense to update existing to `int64_t nVal` or add third with:
```cpp
void CGUIListItem::IncrementProperty(const std::string &strKey, int64_t nVal)
{
  int64_t i = GetProperty(strKey).asInteger();
  i += nVal;
  SetProperty(strKey, i);
}
```

This already use the `int64_t i = GetProperty(strKey).asInteger();` only his given function value not correct.

Reply:
Great idea and much cleaner.